solutions/USACO dec20-abc-bronze.mdx

```diff
@@ -0,0 +1,104 @@
+---
+id: usaco-1059
+source: USACO Bronze 2020 December
+title: Do You Know Your ABCs?
+author: Mrinall Umasudhan
+---
+
+[Official Analysis (C++ and Python)](http://usaco.org/current/data/sol_prob1_bronze_dec20.html)
+
+
+### Implementation
+
+<LanguageSection>
+
+<JavaSection>
+
+```java
+ import java.io.*;
+ import java.util.*;
+
+ public class abc {
```

Comment:
```suggestion
public class abc {
```

Delete the extra space in front of lines. For example, the suggestion above (it's also preferable if you switch to tabs).